### DISCUSSION TITLE
When do Operational Capabilities (OD) turn into System Capabilities (FD)

### DISCUSSION THREAD

**Discussion by napdynamite** [GfSE/SAF-Specification#64] [Q&A] [answered]
Can top level operational capability have sub-capabilities?

For example, the top level capability that a system/solution/product provides to the user is "search and rescue". The sub capabilities can be:
1. search for distress signals on ground.
2. initiate rescue operation. 

both capabilities, on the SOI level, are to pe provided by the system itself (case in point, a UAV). How far down does the operational capability turn into system capability? 

OD-search for distress signals on ground
FD-perform surveillance 
FD-communicate surveillance data to ground units.

**Comment by haarer** [comment, marked as answer]
Am 25. Juni 2024 09:41:42 MESZ schrieb Anas Bin Iftikhar ***@***.***>:
>Can top level operational capability have sub-capabilities?
>
>For example, the top level capability that a system/solution/product provides to the user is "search and rescue". The sub capabilities can be:
>1. search for distress signals on ground.
>2. initiate rescue operation. 
>
>both capabilities, on the SOI level, are to pe provided by the system itself (case in point, a UAV). How far down does the operational capability turn into system capability? 
>
>OD-search for distress signals on ground
>FD-perform surveillance 
>FD-communicate surveillance data to ground units. 
>
>
>
>
>
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/GfSE/SAF-Specification/discussions/64
>You are receiving this because you are subscribed to this thread.
>
>Message ID: ***@***.***>
Yes, operational capabilities can be composed, specialized and can depend on them.
See <https://github.com/GfSE/SAF-Specification/blob/main/viewpoints/Operational-Capability-Definition-Viewpoint.md>

On system Level, system capabilities trace to operational capabilities, and also have similar relationships among them.

On system Level the capability concept is a powerul Tool to develop product lines.

It is a good Idea to structure the system requirements along system capabilities. 
This makes them easier to discuss and maintain. 

Your example is a valid approach, but other shapes of the total capability structure in OD and FD are also possible.

This mostly depends on how far the capabilities haven been analysed and defined in operational Level, and also If there are multiple options of capability provision, e.g. by alternate system approaches/solution classes.

Depending on the operational Story, the drone might communicate directly to the rescue unit, or even control might be transfered to the rescue unit.

Such things must be analysed and clarified already on operational level.
-- 
Gruß,
Alexander
